app/tests/components_tests/test_forms.py

```diff
@@ -190,3 +196,24 @@ def test_image_widget_current_value_in_archive_item_and_display_set_update_forms
         .pk
         == user_upload.pk
     )
+
+
+@pytest.mark.parametrize(
+    "component_interface_kind",
+    [kind for kind in INTERFACE_TYPE_JSON_EXAMPLES.keys()],
+)
+@pytest.mark.parametrize("store_in_db", [True, False])
+@pytest.mark.django_db
+def test_interface_form_field_help_text_example_download_link(

```

Comment:
I think you could:
- move this to a dedicated `test_fields.py`, create it if it doesn't already exists
- not use the django_db but rather pass user=None, this speeds things up a lot
- add a negative test: i.e. add an case where there should _not_ be an example

Reply:
All points were taken except:
> not use the django_db but rather pass user=None, this speeds things up a lot

There are factories for component interface creation which need access to the database to create them.

Maybe I am mistaken, can you please clarify that point and if it is meant for this type of tests?

Reply:
The factories for the component interface creation are indeed DB heavy. However, you can also call them in create object only mode, not having to hit the database. 

There might be another gotcha keeping us from making thist test database free, but not having to hit the database means a one (or maybe two) magnitudes quicker testing.

Reply:
okay got it, I look into it to avoid calling the database

Reply:
I have been trying today to achieve this in multiple ways. For model factories, by calling `build()` on the factory class, it is possible to create an instance without hitting the database. However, `InterfaceFormField` still needs access to the database since it calls `filter_by_permission` and that requires accessing the database through the `guardian` package.

The test fundamentally tests database-backed functionality, so mocking or using pure objects would not provide the same level of integration testing that validates the form field works correctly with the actual database models and relationships.

Lets discuss this further on Monday.

Reply:
Done